templates/opps-list-widget.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * This template is used to display the list of flexible volunteer opportunities
```

Comment:
**Pull request updated**:

Great catch! I've updated the comment to show that the template is also used to display one-time opportunities.